tests/rptest/tests/openmessaging_benchmark_test.py

```diff
@@ -0,0 +1,30 @@
+# Copyright 2020 Vectorized, Inc.
+#
+# Use of this software is governed by the Business Source License
+# included in the file licenses/BSL.md
+#
+# As of the Change Date specified in that file, in accordance with
+# the Business Source License, use of this software will be governed
+# by the Apache License, Version 2.0
+
+import statistics
+
+from rptest.tests.redpanda_test import RedpandaTest
+from rptest.services.cluster import cluster
+from rptest.services.openmessaging_benchmark import OpenMessagingBenchmarkWorker, OpenMessagingBenchmark
+
+
+class OpenBenchmarkTest(RedpandaTest):
+    def __init__(self, ctx):
+        self._ctx = ctx
+        super(OpenBenchmarkTest, self).__init__(test_context=ctx,
+                                                num_brokers=3)
+
+    @cluster(num_nodes=6)
+    def test_sleep(self):
+        workers = OpenMessagingBenchmarkWorker(self._ctx, num_workers=2)
```

Comment:
Are the workers ever used independently of the benchmark service? If not, you might consider starting those workers from within the benchmarking service just to keep the usage of it simpler.